src/main/java/frc/robot/RobotContainer.java

```diff
@@ -95,17 +116,25 @@ public RobotContainer()
 
     private void configureButtonBindings()
     {
-        CmdIntakeStartIntake   startIntake         = new CmdIntakeStartIntake(_intake);
-        CmdIntakeReverseIntake reverseIntake       = new CmdIntakeReverseIntake(_intake);
-        CmdNotepathStartFeed   startNotepathFeed   = new CmdNotepathStartFeed(_notepath);
-        CmdNotepathReverseFeed reverseNotepathFeed = new CmdNotepathReverseFeed(_notepath);
+        CmdIntakeStartIntake     startIntake         = new CmdIntakeStartIntake(_intake);
+        CmdIntakeReverseIntake   reverseIntake       = new CmdIntakeReverseIntake(_intake);
+        CmdNotepathStartFeed     startNotepathFeed   = new CmdNotepathStartFeed(_notepath);
+        CmdNotepathReverseFeed   reverseNotepathFeed = new CmdNotepathReverseFeed(_notepath);
+        CmdShooterFlywheelShoot  flipShoot           = new CmdShooterFlywheelShoot(_shooterFlywheel, 6, 10);
+        CmdShooterFlywheelShoot  straightShoot       = new CmdShooterFlywheelShoot(_shooterFlywheel, 8, 8);
+        CmdShooterBedSetBedAngle setBedLow           = new CmdShooterBedSetBedAngle(_shooterBed, new Rotation2d(Math.PI / 6));
+        CmdShooterBedSetBedAngle setBedHigh          = new CmdShooterBedSetBedAngle(_shooterBed, new Rotation2d(Math.PI / 4));
 
         _drive.setDefaultCommand(DriveCommands.joystickDrive(_drive, () -> -_joystick.getY(), () -> -_joystick.getX(), () -> -_joystick.getZ()));
 
         _controller.y().whileTrue(startNotepathFeed);
         _controller.x().whileTrue(reverseNotepathFeed);
         _controller.a().whileTrue(startIntake);
         _controller.b().whileTrue(reverseIntake);
+        _controller.leftBumper().onTrue(setBedLow);
+        _controller.rightBumper().onTrue(setBedHigh);
+        _controller.back().whileTrue(flipShoot);
+        _controller.start().whileTrue(straightShoot);
     }
```

Comment:
Just an FYI, I rebound the controls to play nicely with everything else we have going on now